pkg/ccl/backupccl/backup_test.go

```diff
@@ -673,7 +673,7 @@ func TestBackupAndRestoreJobDescription(t *testing.T) {
 		[][]string{
 			{fmt.Sprintf("BACKUP TO ('%s', '%s', '%s')", backups[0].(string), backups[1].(string),
 				backups[2].(string))},
-			{fmt.Sprintf("BACKUP INTO '%s' IN ('%s', '%s', '%s')", full1, collections[0],
+			{fmt.Sprintf("BACKUP INTO '%s' IN ('%s', '%s', '%s')", unresolvedFullSubdir, collections[0],
```

Comment:
Since the job record will look different, I wonder how backportable this patch is.

Reply:
see above: if we keep Subdir as is and use a new flag for if we expect a backup to be there or not I think the backwards compat is fine.